### PR TITLE
chore: add common launch params to tensorboard API [DET-4214]

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

add `config`, `files`, and `template_name` parameters.

https://determinedai.atlassian.net/browse/DET-4094 depends on this.

## Test Plan

- tested by using the generated SDK to launch a tensorboard and checking that the added parameters carry over to commandParams (available in api-bindings repo)

## Checklist

- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: master/internal/command/tensorboard_manager.go [hunk 1]
```diff
@@ -158,6 +158,12 @@ func (t *tensorboardManager) processTensorboardRequest(
 	}
 	ctx.Log().Infof("created tensorboard %s", a.Address().Local())
 	summary := summaryResponse.Get().(summary)
+	// REMOVEME
```
Comment (hamidzr) [inline master/internal/command/tensorboard_manager.go:161]: remove

File: proto/src/determined/util/v1/util.proto [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+message File {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: {
+      required: [ "path", "type", "content", "mtime", "mode", "uid", "gid" ]
+    }
+  };
+  // Path to the file or directory.
+  string path = 1;
+  // File type.
+  int32 type = 2;  // FIXME We really only want a single byte.
```
Comment (stoksc) [inline proto/src/determined/util/v1/util.proto:20]: Fix or remove this FIXME before merging?

File: master/internal/api_tensorboard.go [hunk 3]
```diff
@@ -11,12 +13,31 @@ import (
...
+	for _, file := range files {
+		item := archive.Item{
+			Path:     file.Path,
+			Type:     byte(file.Type),
+			FileMode: os.FileMode(file.Mode),
+			Content:  file.Content,
+			UserID:   int(file.Uid),
+			GroupID:  int(file.Gid),
+		}
+		item.ModifiedTime = archive.UnixTime{Time: time.Unix(file.Mtime, 0)}
```
Comment (stoksc) [inline master/internal/api_tensorboard.go:35]: Nit style, pull this into above.

**Review by stoksc — APPROVED** [review]
lgtm 👍

**Pull request merged**